File: atinject-integration/src/main/resources/webapp/scripts/atinject/latency/PingResponse.js
define(["atinject/WebSocketResponse"],
function(WebSocketResponse){
    function PingResponse() {
        this._class = "org.atinject.core.latency.dto.PingResponse"
    };

    PingResponse.prototype = new WebSocketResponse();

        PingResponse.prototype.getTime = function() {
        return this.time;
    };

    PingResponse.prototype.setTime = function(time) {
        this.time = time;
        return this;
    };


    
    return PingResponse;
});

File: atinject-core-latency/src/main/java/org/atinject/core/latency/dto/PingResponse.java
package org.atinject.core.latency.dto;

import org.atinject.core.websocket.dto.WebSocketResponse;

public class PingResponse extends WebSocketResponse {

    private static final long serialVersionUID = 1L;

    private long time;

    public long getTime() {
        return time;
    }

    public PingResponse setTime(long time) {
        this.time = time;
        return this;
    }    
}
